test/e2e_env/universal/observability/meshtrace.go

```diff
@@ -64,10 +75,25 @@ func PluginTest() {
 			}
 			// then traces are published
 			return obsClient.TracedServices()
-		}, "30s", "1s").Should(Equal([]string{
+		}, "30s", "1s").Should(ContainElements([]string{
```

Comment:
Why do we want to change it? Might there be more `TracedServices` than the 3 mentioned below?